app/_locales/en/messages.json

```
@@ -169,6 +169,12 @@
   "advancedBaseGasFeeToolTip": {
     "message": "When your transaction gets included in the block, any difference between your max base fee and the actual base fee will be refunded. Total amount is calculated as max base fee (in GWEI) * gas limit."
   },
+  "advancedGasFeeDefaultOptIn": {
+    "message": "Use the “new values” and advanced setting as default."
```

I'm not seeing this copy the designs? And personally I think it's a bit confusing. Maybe something like?: 
```suggestion
    "message": "Set the currently selected values as defaults when using advanced settings."
```

---

The last image for the advanced gas modal in this [Frame](https://www.figma.com/file/DFlkrM9JRyP0hDrd4ykMZW/1559-Gas-Controls---v2.1?node-id=610%3A7104) shows that copy. It is displayed when the checkbox is not selected. 

@jakehaugen Can I get your input on updating the copy?